app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
+  end
```

Comment:
```suggestion
def full_title(page_title = nil)
  base_title = "Ruby on Rails Tutorial Sample App"
  [page_title, base_title].compact.join(" | ")
end
```

Reply:
Take a look at what `compact` does:

https://devdocs.io/ruby~2.4/array#method-i-compact

And join:

https://devdocs.io/ruby~2.4/array#method-i-join